lib/cli-engine/config-array-factory.js

```diff
@@ -623,8 +623,11 @@ class ConfigArrayFactory {
      * @private
      */
     _normalizeConfigData(configData, ctx) {
+
         validateConfigSchema(configData, ctx.name || ctx.filePath);
-        return this._normalizeObjectConfigData(configData, ctx);
+        const clonedConfig = JSON.parse(JSON.stringify(configData));
```

Comment:
nits(non-blocker): better to have a comment explaining why it is needed. 😄

Reply:
should I add a link to https://github.com/eslint/eslint/pull/13034#discussion_r420688527 ?.

Reply:
a link to the original issue, and the above comment?

Reply:
ok cool 👍